### PR TITLE
refactor: seperate zxcvbn dependencies

### PR DESCRIPTION
**Pull request by HosseinSalmanian**
zxcvbn implementation  of IPasswordStrengthMeterService moved to second entrypoint in order to remove including zxcvbn modules into final bundle when client has custom implementation of IPasswordStrengthMeterService.

### CODE REVIEW

File: projects/password-strength-meter/src/lib/password-strength-meter.component.ts [hunk 1]
```diff
@@ -8,7 +8,8 @@ import {
   HostBinding,
 } from '@angular/core';
 
-import { IPasswordStrengthMeterService } from './password-strength-meter.service';
+import { IPasswordStrengthMeterService } from './password-strength-meter-service';
+
```
Comment (antoantonyk) [inline projects/password-strength-meter/src/lib/password-strength-meter.component.ts:12]: Unwanted new line

File: projects/password-strength-meter-showcase/src/app/custom-psm-service.service.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { Injectable } from '@angular/core';
-import { IPasswordStrengthMeterService } from 'projects/password-strength-meter/src/public-api';
+import { IPasswordStrengthMeterService } from 'angular-password-strength-meter';
```
Comment (antoantonyk) [inline projects/password-strength-meter-showcase/src/app/custom-psm-service.service.ts:2]: nice

**Review by antoantonyk — COMMENTED** [review]
LGTM 👍 . Could you please merge the latest changes?  This is will be a breaking change. We can release it as a new version? v7

File: projects/password-strength-meter/src/lib/password-strength-meter.component.ts (hunk 1 above)
Comment (HosseinSalmanian) [reply projects/password-strength-meter/src/lib/password-strength-meter.component.ts:12]: I've removed the unnecessary new line.

**Comment by HosseinSalmanian** [comment]
Hi @antoantonyk 
I've resolved the merge conflicts.

**Comment by OneHoopyFrood** [comment]
Would be nice to see this merged.

File: package.json [hunk 3]
```diff
@@ -35,8 +35,6 @@
     "@angular/platform-browser": "^15.2.9",
     "@angular/platform-browser-dynamic": "^15.2.9",
     "@angular/router": "^15.2.9",
-    "@zxcvbn-ts/core": "^2.2.1",
-    "@zxcvbn-ts/language-en": "^2.1.0",
```
[the remark this answers is not in the document]
Comment (antoantonyk) [reply package.json:39]: or do we need to use `npm link`?

[the remark this answers is not in the document]
Comment (antoantonyk) [reply package.json:39]: some details -https://github.com/angular/angular/issues/43058

**Pull request merged**